docs/streamtypes/caip-10-link/api.md

```diff
@@ -65,20 +46,21 @@ The final argument to `Caip10Link.fromAccount` is an instance of [`CreateOpts`](
 | `sync`        | optional    | enum             | Controls behavior related to syncing the current link state from the network | SyncOptions.PREFER_CACHE |
 | `syncTimeoutSeconds` | optional    | number            | How long to wait to hear about the current state of the link from the network | 3 |
 
-[:octicons-file-code-16: API reference](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_common.createopts-1.html){:target="_blank"}
+!!! warning ""
 
+    By default `Caip10Link.fromAccount()` will try to query the network for the current stream state. If you know that this is the first time a Caip10Link has been created for this blockchain address, and you wish to avoid waiting on a response from the network unnecessarily, you can set `syncTimeoutSeconds` to 0 in the `opts` argument.
 
-## Associate a DID with a Caip10Link
-Use the [`link.setDid()`](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_stream_caip10_link.caip10link-1.html#setdid){:target="_blank"} method to update the DID associated with a given Caip10Link
+
+### **Set DID to Caip10Link**
```

Comment:
the grammar structure here feels weird.  Are you setting a did *to* a caip10link? It feels more like you're setting a caip10link to a did.  I actually like the original wording "Associate a DID with a Caip10Link"

Reply:
I was basing it on the API method which you're setting a DID. This makes sense to me since you're first creating the stream, then adding a DID, and you can remove a DID/set a new DID.